packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -90,10 +90,16 @@ export class MyAlgoWalletSession {
 	}
 
 	// https://connect.myalgo.com/docs/interactive-examples/Connect
```

Comment:
You can add this link in the description you have added. Something like `for more info visit: <url>`

Reply:
oh yes, thanks will do that